rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Book and person could be writable too, please change it. 🐱

Reply:
Issue fixed

Reply:
![rental](https://user-images.githubusercontent.com/57562869/136303008-2a4ca5cb-29a4-4c56-964f-111a8b5fcad1.png)
- Add the setter methods for book and person